src/api/email.rs

```diff
@@ -1,8 +1,6 @@
 //! You'll find in email sending related endpoints.
 mod send_email;
 mod send_email_batch;
-mod send_email_with_template;
 
 pub use send_email::*;
 pub use send_email_batch::*;
-pub use send_email_with_template::*;
```

Comment:
Hello! I am not sure if this is on purpose, but it seems like this module is no longer included in the code. It does seem like `send_email_with_template.rs` is still in the repo here https://github.com/pastjean/postmark-rs/blob/main/src/api/email/send_email_with_template.rs

I had some downstream code that used `SendEmailWithTemplate` that no longer compiles with 0.9.0. I cannot figure out an alternative. Seems like this might have been an accident?